### DISCUSSION TITLE
Session Names

### DISCUSSION THREAD

**Discussion by MarathonMax** [sienori/Tab-Session-Manager#1206] [Ideas]
It may have already been proposed, but I would like to name my sessions or groups. You see, I use this extension to allow me to have different windows, one per client (I am a marketing consultant for tech firms). Right now, Tab Session Manager names the sessions per the first tab of a window, for example, "Inbox" if Gmail is the first tab. I would like to name each one individually and manually. Tks. Ok ok, I see that this feature has been requested already [here](https://github.com/sienori/Tab-Session-Manager/discussions/809).

**Comment by arseniigorkin** [comment]
Vote up!
